internal/app/machined/internal/server/v1alpha1/v1alpha1_server.go

```diff
@@ -149,7 +149,7 @@ func (s *Server) ApplyConfiguration(ctx context.Context, in *machine.ApplyConfig
 		}
 
 		go func() {
-			if err := s.Controller.Run(runtime.SequenceApplyConfiguration, in); err != nil {
+			if err := s.Controller.Run(ctx, runtime.SequenceApplyConfiguration, in); err != nil {
```

Comment:
we shouldn't use `grpc` request context for it, as it will abort sequence when request is done, and it's done almost immediately. this should use `context.Background()` (same in other places when we launch sequence in a goroutine from grpc context)

Reply:
yeah, I got hit by that already. Fixing :)